mllib/src/main/scala/org/apache/spark/ml/util/MetadataUtils.scala

```diff
@@ -41,6 +41,24 @@ private[spark] object MetadataUtils {
     }
   }
 
+  /**
+   * Examine a schema to identify the number of features in a vector column.
```

Comment:
The same method is also added in PR for RobustScaler, I tend to merge that first, and rebase this PR.